### ISSUE TITLE
How to get the position map of

### ISSUE THREAD

**Issue by Luh1124** [csbhr/FFHQ-UV#76]

**Comment by Luh1124** [comment]
I'm trying to use the function “unwrap_vertex_to_uv” to unwrap the **spatial position coordinates** of the vertices to UV, but I'm encountering issues **with bilinear interpolation and resampling based on the vertex UV coordinates**. Can you give me a hint on how to **achieve the correct results for the position map unwrapping?**

**Comment by Luh1124** [comment]
my code and results：
    def inverse_vertex_from_vertex_uv(self, vertex_uv):
        vtx_vt = (self.facemodel.vtx_vt / 512) * 2 - 1 # [-1, 1] [20481 2]
        vertex_uv = torch.from_numpy(vertex_uv).unsqueeze(0).to(self.device) # [1, H, W, 2]
        inverse_vertex = F.grid_sample(vertex_uv.permute(0, 3, 1, 2), vtx_vt.unsqueeze(0).unsqueeze(0), mode='bilinear') # [1, 3, 1, 20481]
        inverse_vertex = inverse_vertex.permute(0, 2, 3, 1).squeeze(0).detach().cpu().numpy()
        return inverse_vertex  # [1, 20481, 3]
        
        
simple visualization（"Ignore numerical overflow." ）：
![image](https://github.com/csbhr/FFHQ-UV/assets/45451989/d22e026a-c50d-408f-87a7-fc63c623d512)
